---
title: "Reexamining populations of beluga in the Hudson Bay-Strait Complex and assessing the impact on harvests in Nunavik and Sanikiluaq management units"
output: github_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

__Main author:__  Genevi√®ve J. Parent  
__Affiliation:__  Fisheries and Oceans Canada (DFO)   
__Group:__        Laboratory of genomics   
__Location:__     Maurice Lamontagne Institute  
__Affiliated publication:__  Fisheries and Oceans Canada, Canadian Science Advisory Secretariat   
__Contact:__      e-mail: genevieve.parent@dfo-mpo.gc.ca | tel: (418) 732-4329


- [Objective]
- [Summary] 
- [Status]
- [Supplementary material]
  + [Table S1] 
  + [Table S2]
  + [Table S3] 
  + [Table S4]


## Objective
Sequencing all available beluga samples collected between 1982 and 2021 to 1) re-examine the distinctiveness of populations in the Hudson Bay-Strait Complex by comparing short and long mtDNA haplotypes and 2) estimate the proportional contribution of each of the newly-defined genetic groups to the harvest conducted in Nunavik and Sanikiluaq management units.


## Summary
Belugas from the Hudson Bay-Strait Complex are harvested by hunters from Nunavik and Nunavut communities. In past studies, a genetic mixture analysis (GMA) was used to determine the contribution of animals from the Western Hudson Bay (WHB) and Eastern Hudson Bay (EHB) populations to harvests in the different management units. The population definition of WHB and EHB relied on short haplotypes from the mitochondrial (mt) DNA control region. However, studies with long haplotypes have shown that four populations could be identified in the Hudson Bay-Strait Complex. Here, we aim to 1) revisit the definition of populations within this area by resequencing and comparing short and long haplotypes for the mtDNA control region of 2861 belugas, and 2) update the relative contribution of the newly-defined reference groups in management units using a GMA. Long haplotypes confirmed the existence of four previously defined populations, WHB, EHB, James Bay (JAM), and Cumberland Sound (CSB). They also allowed for the identification of a fifth population in the Belcher Islands (BEL). For the second objective, we tested the validity of the five populations using a leave-one-out approach and observed a high rate of erroneous assignments between EHB and BEL populations. Misassignments were due to shared genetic matrilineages and possible admixture during summer. Assignments were improved when EHB and BEL populations were combined in as a single reference named BEL-EHB stock. The GMA with the four new reference groups (WHB, JAM, CSB populations and BEL-EHB stock) led to higher proportions of the BEL-EHB stock in the fall harvest in Hudson Strait (5.6%) and Northeastern Hudson Bay (14.9%), and year-round in Sanikiluaq (3.3 to 61.2%), compared to previous studies using only two reference groups. Changes in reference groups, and not resequencing, likely caused most of the variation in the proportional estimate of the BEL-EHB stock in management units. Results from this study have increased the accuracy of the proportional contribution from reference groups to the harvest by the Nunavik and Nunavut (Sanikiluaq) communities.


## Status
In press.


## Supplementary material
This page has for goal to provide data (in tables) associated to the publication.


### Table S1
Metadata for each beluga from this study (N = 2861)


### Table S2
Sequence library of short control region in the mtDNA of belugas from the Hudson Bay-Strait Complex


### Table S3
Sequence library of long control region in the mtDNA of belugas from the Hudson Bay-Strait Complex


### Table S4
Long haplotype frequencies per summering area


*This github readme was generated by knitting the README.Rmd from
Rstudio.*